.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-merge-conflict
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+
+-   repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.31.0
+    hooks:
+    -   id: yapf
+
+-   repo: https://gitlab.com/PyCQA/flake8
+    rev: 3.8.4
+    hooks:
+    -   id: flake8
+        args: [--count]
+
+-   repo: local
```

Comment:
pytest slow are slow tests. They take take a long time to run (more than 10 mins). This can be removed.

Reply:
Addressed in 996d526